<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrontPageWidget</class>
 <widget class="QWidget" name="CFrontPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pCreatePBtn">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pReadPBtn">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pUpdatePBtn">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pDeletePBtn">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="m_pTableView">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pCreatePBtn</tabstop>
  <tabstop>m_pReadPBtn</tabstop>
  <tabstop>m_pUpdatePBtn</tabstop>
  <tabstop>m_pDeletePBtn</tabstop>
  <tabstop>m_pTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
